.gitignore

```diff
@@ -34,3 +34,9 @@ ironic-inspector-auth-config
 ironic-rpc-auth-config
 HTTP_BASIC_HTPASSWD
 ironic-deployment/overlays/temp
+
+# macOS generated
+.DS_Store
+
+# jetbrains generated
+.idea
```

Comment:
Please put ignore rules for your own tools in your own global gitignore file instead (`~/.config/git/ignore` on Linux, I'm not sure about MacOS).